R/discog.R

```diff
@@ -7,4 +7,9 @@
 #'   Sharla's music collection.
 #' @examples
 #' length(discog)
+#' discog %>%
+#' head(2) %>%
+#' str(max.level = 2)
+#' @note Yao Nie from Tidyverse Developer Day, July 8, 2019
+#'   Toulouse, France
```

Comment:
I reworked this to *not* use the pipe operator `%>%`, because this package itself does not make it available. It's a surprising constraint that you just have to get used to: it's best to make the examples runnable using only the package itself.